app/assets/javascripts/no_sunet.js

```diff
@@ -0,0 +1,43 @@
+$(document).on('click', '#no-sunet-button', function(){
+  setTimeout(function(){
+      $('#request_user_attributes_library_id').focus();
+  }, 1);
+
+  $('form').on('keypress', function(e) {
+    if(e.which == 13) {
+      $('.form-group input').checkForm(e);
+    }
+  });
+
+  $('.form-control').on('mouseenter mouseleave focus blur',function(e){
+    $('.form-group input').checkForm(e);
+  });
+
+  $('#no-sunet-submit').click(function(e) {
+    if ($('.form-group input').checkForm(e)){
+      $('form').submit();
+    }
+  });
+});
+
+$.fn.checkForm = function(e) {
+  var empty = true;
+
+  $(this).each(function() {
```

Comment:
I'm worried that this code is brittle (one of the *code smells* is the blacklist of elements by ID).  There are several things somebody else could do to the form that would break this functionality in unexpected ways. I would be more than happy to chat about ways to accurately identify which inputs need to be validated.